src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework.UnitTests/HandlerProviderTests.cs

```diff
@@ -82,6 +82,22 @@ public void GetRegisteredMethods(bool supportsGetRegisteredServices)
             r => Assert.Equal(TestParameterlessNotificationHandler.Name, r.MethodName));
     }
 
+    [Fact]
+    public void GetMethodHandler_LanguageHandlers()
+    {
+        var handlerProvider = new TestHandlerWithLanguageProvider(providers: new[]
+        {
+            (TestXamlLanguageHandler.Metadata, TestXamlLanguageHandler.Instance),
+            (TestDefaultLanguageHandler.Metadata, TestDefaultLanguageHandler.Instance),
+        });
```

Comment:
Nit: 

```suggestion
        var handlerProvider = new TestHandlerWithLanguageProvider(providers: [
            (TestXamlLanguageHandler.Metadata, TestXamlLanguageHandler.Instance),
            (TestDefaultLanguageHandler.Metadata, TestDefaultLanguageHandler.Instance),
        ]);
```

Reply:
Nice. Collection initializers finally work in Roslyn. I was trying to use them earlier in the PR but they would cause build errors back then.